ci_framework/roles/devscripts/vars/main.yml

```diff
@@ -34,7 +34,7 @@ cifmw_devscripts_rhsm_enabled_repos: false
 cifmw_devscripts_config_defaults:
   working_dir: "/home/dev-scripts"
   openshift_release_type: "ga"
-  openshift_version: "{{ cifmw_devscripts_ocp_version | default('4.13.15') }}"
+  openshift_version: "{{ cifmw_devscripts_ocp_version | default('4.13.18') }}"
```

Comment:
This change is going to affect all tests. Are you sure with that?

Reply:
Yes. I have run multiple tests and found this to be more stable.

Reply:
OK, we need to announce it more broadly to the whole CI team.